src/orange/OrangeData.hh

```diff
@@ -217,18 +221,38 @@ struct TransformRecord
 //---------------------------------------------------------------------------//
 /*!
  * Scalar data for a single "unit" of volumes defined by surfaces.
+ *
+ * If using a HEP geometry implementation and this unit represents multiple
+ * parent/child levels of the geometry hierarchy (via "inlining" during
+ * construction; see \c g4org::ProtoConstructor):
+ * - Local parent is a map of local to local volume IDs: the value is another
```

Comment:
Specify that both the key and value refer to canonical volumes

Reply:
They refer to local impl volumes, but the "parent" aspect refers to the canonical volumes they represent